### ISSUE TITLE
Insufficient contrast for links on Blog page

### ISSUE THREAD

**Issue by TimJentzsch** [Leafwing-Studios/leafwing-website#232]
## What's wrong?

For the blog links we have `#5ca329` on `#f7f4ed` which doesn't satisfy the WCAG accessibility guidelines for contrast (see [Color ID](https://color-id.timjen.net/?color=rgb(92%2C%20163%2C%2041)&secondary=%23f7f4ed) for more details).

To satisfy WCAG AA, we can take `#457b1f` instead, for WCAG AAA `#2e5215`.
